include/deal.II/matrix_free/shape_info.h

```diff
@@ -150,7 +150,7 @@ namespace internal
       /**
        * Stores the shape values of the 1D finite element evaluated on all 1D
        * quadrature points in vectorized format, i.e., as an array of
-       * VectorizedArray<dim>::n_array_elements equal elements. The length of
+       * VectorizedArray<dim>::size() equal elements. The length of
        * this array is <tt>n_dofs_1d * n_q_points_1d</tt> and quadrature
        * points are the index running fastest.
        */
```

Comment:
This comment is actually outdated since we changed the data field to `Number` a while ago. Can you simply delete the statement starting from `in vectorized format` and replace it by `If the Number type is a VectorizedArray, the same value will be broadcast to all lanes.` Change here and in all other places. Note also that this will likely conflict with #9655 so we need to be careful here.

Reply:
I have removed the changes form `shape_info` so that we don't get any merge issues.